roles/rsyslog/roles/input_roles/ovirt/defaults/main.yaml

```diff
@@ -153,6 +161,28 @@ rsyslog_conf_ovirt_formatting:
           }
           {% endif %}
 
+          if not ($syslogtag startswith "ovirt" or $syslogtag startswith "collectd") then {
+              # Log all kernel messages to the console.
+              # Logging much else clutters up the screen.
+              #kern.*                                                 /dev/console
+              # Log anything (except mail) of level info or higher.
+              # Don't log private authentication messages!
+              *.info;mail.none;authpriv.none;cron.none                /var/log/messages
+              # The authpriv file has restricted access.
+              authpriv.*                                              /var/log/secure
+              # Log all the mail messages in one place.
+              mail.*                                                  -/var/log/maillog
+
+              # Log cron stuff
+              cron.*                                                  /var/log/cron
+              # Everybody gets emergency messages
+              *.emerg                                                 :omusrmsg:*
+              # Save news errors of level crit and higher in a special file.
+              uucp,news.crit                                          /var/log/spooler
+              # Save boot messages also to boot.log
+              local7.*                                                /var/log/boot.log
+        }
+
```

Comment:
@richm @didib @pcahyna Please review this patch.
I have a problem in the design with this. If I use the default rsyslog.conf the collectd logs go to /var/log/messages.
I needed to move the config to this section to avoid getting the collectd logs getting to /var/log/messages.
But this is really not a generic fix. 
I can add it here for now but would prefer to replace it to a more generic solution.

Reply:
@sradco, I'd think what is implemented in https://github.com/sradco/logging/blob/updatedefaultconf/roles/rsyslog/roles/input_roles/ovirt/defaults/main.yaml#L164-L184 is not input, but action/output.

(Sorry, I haven't tested) but the code written in the old format could be interpreted to the current syntax as follows.  They are either omfile or omusrmsg action.  So, I guess you could place it in the output roles?  And if its input is not from viaq, they don't have $syslogtag (and $syslogfacility or $syslogseverity) at all?  (@richm, please correct me if I'm wrong).    Then, it'd be safe to have a default output role in output_roles and include it?

```
if strlen($syslogtag) == 0 then {
    if $facility == "kern" then {
        action(type="omfile" template="RSYSLOG_FileFormat" file="/dev/console")
    } else if $facility == "mail" then {
        action(type="omfile" template="RSYSLOG_FileFormat" file="/var/log/maillog")
    } else if $facility == "authpriv" then {
        action(type="omfile" template="RSYSLOG_FileFormat" file="/var/log/secure")
    } else if $facility == "cron" then {
        action(type="omfile" template="RSYSLOG_FileFormat" file="/var/log/cron")
    } else if $facility == "local7" then {
        action(type="omfile" template="RSYSLOG_FileFormat" file="/var/log/boot.log")
    } else {
        action(type="omfile" template="RSYSLOG_FileFormat" file="/var/log/messages")
    }
 
    if $severity == "emerg" then {
        action(type="omusrmsg" users="*")
    }
    if (($facility == "uucp") or ($facility == "news")) and ($severity == "crit") then {
        action(type="omfile" template="RSYSLOG_FileFormat" file="/var/log/spooler")
    }
}
```

Reply:
@nhosoi Yes. you are correct. I did not add this to the correct location. It should defiantly be in output roles.
logs coming from journal don't have  $syslogtag ?

Reply:
> they don't have $syslogtag (and $syslogfacility or $syslogseverity) at all?

I don't know, but you could easily check.  But if there is an easy way to check system messages like @nhosoi proposes, that would be best.

Reply:
I see that debops do have this rules already here:
https://github.com/linux-system-roles/logging/blob/059d70fe43ca78b497c87f606e1ed133cf373867/roles/rsyslog/roles/input_roles/debops/defaults/main.yaml#L166

How would you differentiate between a case we wants default rsyslog.conf that includes these rules and a case that we other input roles like viaq/ovirt and also need these rules?
We currently have only 1 variable "rsyslog_default" and it is true by default.

I think we can filter by $.logs_collection. If it is no set then go to these rules.

@nhosoi

Reply:
I think that some users of ovirt will want ovirt and collectd logs to go to `/var/log/messages`, and some will not.   It is probably a good idea for ovirt to configure rsyslog by default to not send these logs to `/var/log/messages`.